sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/OperatorStateMetadata.scala

```diff
@@ -89,6 +91,40 @@ case class OperatorStateMetadataV2(
   override def version: Int = 2
 }
 
+object OperatorPropertiesUtils {
+  class OperatorProperties(
```

Comment:
I guess this is not needed any more ? we can just use `TransformWithStateOperatorProperties` ?

Reply:
On I included this so that other Operators could use this in the future, but we can remove it.